tooling/api/src/path.ts

```diff
@@ -28,23 +28,76 @@ import { BaseDirectory } from './fs'
 import { isWindows } from './helpers/os-check'
 
 /**
- * Returns the path to the suggested directory for your app config files.
+ * Alias of {@link appConfigDir} for backwards-compatibility purposes.
+ *
+ * @since 1.0.0
+ */
```

Comment:
keep the original doc comment here and add `@deprecation` notice saying this will be removed in v2 and suggest using the new function